inst/assets/BS4/pkgdown.css

```diff
@@ -392,3 +392,16 @@ div.pb-2.mt-4.mb-2.border-bottom {
   will-change: transform;
   transition: transform 400ms ease;
 }
+
+/* Avoid flash of invisible text */
+body {
+  font-display: swap;
```

Comment:
Wouldn't `fallback` be a little better?  (https://css-tricks.com/almanac/properties/f/font-display/)

Reply:
Not what Google Lighthouse seems to recommend (btw it is something that's in the report but doesn't affect the score) but good compromise indeed.